### PR TITLE
Consistent error messages

### PR DESCRIPTION
**Pull request by ByteBlast**
Work in progress.

### CODE REVIEW

**Comment by ByteBlast** [comment]
I had intended to make the wording more consistent but I changed my mind.

Can you please review and if it looks good, merge, @robdmoore? Thank you.

**Comment by ByteBlast** [comment]
I know you have been busy moving house etc. (congratulations by the way:balloon:) so just to remind you in case you forgot:

We are sitting on version 3.0.0 at the moment. 

When I implement #54 I will kindly ask you to publish version 3.0.0. Please do not publish the new package *yet*.

Cheers.

**Comment by robdmoore** [comment]
When your notifications come through on email your name is "null" FYI. Not sure if deliberate :P

File: TestStack.FluentMVCTesting.Tests/ControllerExtensionsTests.cs [hunk 1]
```diff
@@ -58,13 +58,13 @@ public void Check_for_unexpected_non_existent_temp_data_property()
...
 
             var exception = Assert.Throws<TempDataAssertionException>(() =>
-                _controller.ShouldHaveTempDataProperty(key));
-
-            Assert.That(exception.Message, Is.EqualTo(string.Format(
-                "Expected TempData to have a non-null value with key \"{0}\", but none found.", key)));
-        }
-
-        [Test]
+                _controller.ShouldHaveTempDataProperty(key));
```
Comment (robdmoore) [inline TestStack.FluentMVCTesting.Tests/ControllerExtensionsTests.cs]: Why is the diff so big? Is there whitespace changes?

**Comment by ByteBlast** [comment]
Yeah, it is.

File: TestStack.FluentMVCTesting.Tests/ControllerExtensionsTests.cs (hunk 1 above)
Comment (ByteBlast) [inline TestStack.FluentMVCTesting.Tests/ControllerExtensionsTests.cs]: Think so.

**Comment by robdmoore** [comment]
Happy for you to merge this. Might be worth seeing if you can stop the whitespace changes though?

**Pull request merged**